doc/man3/BIO_f_ssl.pod

```diff
@@ -241,6 +241,16 @@ a client and also echoes the request to standard output.
      exit(1);
  }
 
+ /*
+  * Added BIO_do_accept call: the first BIO_do_accept made the socket bind 
+  *  and listen and in the second BIO_do_accept will do an accept
```

Comment:
remove one blank between "*" and "and"

Reply:
BTW: I am not sure but I would probably add write "and listen, and the second BIO_do_accept will do an accept"
so a comma before the and, and no in because the grammar will not work out this way.